code/modules/cargo/packs.dm

```diff
@@ -1319,6 +1344,15 @@
 	access = ACCESS_CMO
 	contains = list(/obj/item/clothing/under/rank/medical/chief_medical_officer/turtleneck,
 					/obj/item/clothing/under/rank/medical/chief_medical_officer/turtleneck/skirt)
+	crate_name = "chief medical officer turtlenecks"
```

Comment:
Duplicate line.

Reply:
Ah, I see

Reply:
I added that crate_name to the CMO turtleneck crate because it didn't have one before. Consider it a neat little fix to something that was broken before. Also if you check any other crate you will see that they have both a name and a crate_name

Reply:
Fair, alright then